src/style.css

```diff
@@ -0,0 +1,24 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.main_block {
```

Comment:
For this task, there's no need to use classes because all your tags have the same styling and don't differ one from another. You could simply use `h1`, `h2` etc.